hack/tools/tilt-prepare/main.go

```diff
@@ -193,12 +227,19 @@ func tiltResources(ctx context.Context) error {
 		if len(pValues) == 3 && pValues[2] == "debug" {
```

Comment:
Should we drop the debug handling here as well as the `:debug` in the Tiltfile and make it conditional on the existence of the debug field in the given tilt-settings file/struct that is being read by this PR?

Reply:
Yes.
This comes down to handling the configuration directly here instead of partially parsing it in the Tiltfile and then handing information over. So just handling the config here is a lot better.